### ISSUE TITLE
Implement on-hover events and cursor shapes

### ISSUE THREAD

**Issue by siphiuel** [status-im/react-native-desktop#257]
It would be nice to support cursor shape changes over TouchableHighlight, Button, and Image components.

**Comment by vkjr** [comment]
@siphiuel, What will be the use case?

**Comment by vkjr** [comment]
@siphiuel, I think it is not very relevant now, wdyt?

**Comment by siphiuel** [comment]
@vkjr Agree! It's just a nice UI feature enabling better UI discoverability, but certainly not critical.
